tests/__init__.py

```diff
@@ -1,4 +1,105 @@
 """Tests for the nibiru-py package"""
+import dataclasses
+import os
+import unittest
+
 import dotenv
+import pytest
+
+import nibiru
+from nibiru import common
 
 dotenv.load_dotenv()
+
+
+@dataclasses.dataclass
+class TestConfig:
+    LCD_PORT = "1317"
+    GRPC_PORT = "9090"
+    VALIDATOR_MNEMONIC = os.environ["VALIDATOR_MNEMONIC"]
+    ORACLE_MNEMONIC = os.environ["ORACLE_MNEMONIC"]
+    NETWORK_INSECURE = os.environ["NETWORK_INSECURE"]
+    CHAIN_ID = os.environ["CHAIN_ID"]
+    HOST = os.environ["HOST"]
+
+
+CONFIG = TestConfig()
+
+
+class ModuleTest(unittest.TestCase):
+    def validate_tx_output(self, tx_output: dict):
+        """
+        Ensure the output of a transaction have the fields required and that the raw logs are properly parsed
+
+        Args:
+            tx_output (dict): The output of a transaction in a dictionary
+        """
+
+        self.assertIsInstance(tx_output, dict)
+        self.assertCountEqual(
+            tx_output.keys(),
+            [
+                'height',
+                'txhash',
+                'data',
+                'rawLog',
+                'logs',
+                'gasWanted',
+                'gasUsed',
+                'events',
+            ],
+        )
+        self.assertIsInstance(tx_output["rawLog"], list)
+
+
+def get_network() -> nibiru.Network:
+    return nibiru.Network(
+        lcd_endpoint=f'http://{CONFIG.HOST}:{CONFIG.LCD_PORT}',
+        grpc_endpoint=f'{CONFIG.HOST}:{CONFIG.GRPC_PORT}',
+        chain_id=CONFIG.CHAIN_ID,
+        fee_denom='unibi',
+        env="local",
+    )
```

Comment:
The `get_network` function isn't needed because you can use the class method `nibiru.Network.devnet` directly in the pytest fixture below.

Reply:
Best practice here is to keep all fixtures in `conftest.py`. This makes fixtures available across all tests. See here:
https://github.com/NibiruChain/nibiru-py/pull/65/files#diff-a31c7ed5d35f5ed8233994868c54d625b18e6bacb6794344c4531e62bd9dde59R9-R23